src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.data.mongodb.uri=mongodb://localhost:27017/trainingPoc
```

Comment:
Are you able to connect to MongoDb when running this application in a docker container?

Reply:
while commiting this I have't tried the application running in docker, so we have to change the uri method to host method to run an application in docker.